libraries/AP_HAL_ChibiOS/Util.cpp

```diff
@@ -78,6 +77,44 @@ void* Util::try_alloc_from_ccm_ram(size_t size)
     return ret;
 }
 
+#ifdef ENABLE_SCRIPTING
```

Comment:
This should be # ENABLE_HEAP and enabling scripting enables the heap.

Reply:
Done.